src/components/control-bar/control-bar-items/MeltMenuItems.svelte

```diff
@@ -0,0 +1,27 @@
+<!--
+  (c) 2023, Center for Computational Thinking and Design at Aarhus University and contributors
+
+  SPDX-License-Identifier: MIT
+ -->
+
+<script lang="ts">
+  import { createDropdownMenu } from '@melt-ui/svelte';
+  import { scale } from 'svelte/transition';
+  import { quintOut } from 'svelte/easing';
+
+  export let dropdownMenu: ReturnType<typeof createDropdownMenu>;
+  const { menu } = dropdownMenu.elements;
+</script>
+
+<div
+  use:menu
+  {...$menu}
+  {...$$restProps}
+  transition:scale={{
+    duration: 200,
+    start: 0.5,
+    easing: quintOut,
+  }}
```

Comment:
Replace MenuTransition component by embedding this kind of transition. For some reasons `MenuTransition` was not working when I wrapped the MenuItems with it

Reply:
I had a similar problem with one of the Transition components in the standard dialog getting out of sync with $open. Don't know why. Easier to remove than to investigate at this point.